roles/galaxy-route/defaults/main.yml

```diff
@@ -4,7 +4,7 @@
 #
 route_host: ''
 
-hostname: '{{ deployment_type }}.example.com'
+hostname: ''
```

Comment:
What's the reason for changing this ?

I think this will break deployment with ingress enabled because ingresses hostname can't be empty

Reply:
I'm trying to match it to the awx and eda-server operators

https://github.com/ansible/awx-operator/blob/devel/roles/installer/defaults/main.yml#L80
https://github.com/ansible/eda-server-operator/blob/main/roles/eda/defaults/main.yml#L150

Reply:
that's a little bit different in awx/eda because the host field is only set if the `hostname` variable isn't empty

The current change will likely (because I didn't test this) generate and error because the host field is empty.

I would recommend to remove that change from this PR and create a dedicated one for that (in addition to the if statement as below)

https://github.com/ansible/awx-operator/blob/devel/roles/installer/templates/networking/ingress.yaml.j2#L37-L39
https://github.com/ansible/eda-server-operator/blob/main/roles/eda/templates/eda-ui.ingress.yaml.j2#L30-L32

Reply:
done, will address in a future PR